torch/_inductor/kernel/mm_common.py

```diff
@@ -53,6 +53,34 @@ def mm_configs():
         triton.Config(
             {"BLOCK_M": 32, "BLOCK_N": 32, "BLOCK_K": 16}, num_stages=1, num_warps=2
         ),
+        # good for int8
```

Comment:
Let's not just add all these configs to `mm_configs`, as it will significantly increase the time to run `max_autotune=True`.